koku/api/common/pagination.py

```diff
@@ -195,12 +195,6 @@ class ForecastListPaginator(ListPaginator):
 
     default_limit = DateHelper().this_month_end.day
 
-
-class AWSForecastListPaginator(ListPaginator):
-    """A paginator that applies a default limit based on days in month."""
-
-    default_limit = DateHelper().this_month_end.day
-
     def __init__(self, data_set, request, cost_type):
```

Comment:
`cost_type` should be optional.
```suggestion
    def __init__(self, data_set, request, cost_type=None):
```